apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.types.ts

```diff
@@ -0,0 +1,19 @@
+import z from 'zod';
+
+const FormCheckboxSchema = z
+  .enum(['on', ''])
+  .optional()
+  .transform((v) => v === 'on');
```

Comment:
We already have `BooleanFromCheckboxSchema` in `@prairielearn/zod`, let's use that.